### DISCUSSION TITLE
To do list

### DISCUSSION THREAD

**Discussion by bendichter** [EthoML/VAME#6] [General]
Python package:
* [ ] GitHub configuration
    * [x] Set up Issues and Actions
    * [x] Basic tests #12 
    * [x] Set up continuous integration (cross OS)
    * [x] protect master branch, require PR and tests to pass
* [ ] Dependencies
    * [x] Update from old setup.py dependency configuration to pyproject.tomlhttps://github.com/EthoML/VAME/pull/3
    * [x] Fix broken dependencies https://github.com/EthoML/VAME/pull/3
    * [x] Upgrade Python to 3.11 https://github.com/EthoML/VAME/pull/3
    * [x] Deploy on pypi #18 
    * [ ] Deploy on conda
    * [x] Continuous deployment for Python library #18 
    * [x] Use semantic versioning #3 
* [ ] Testing
    * [ ] Create full unit test suite  
    * [x] use codecov to get test coverage (add badge) #18 
* [ ] Documentation
    * [ ] Produce a self-contained demo
    * [x] Improve README https://github.com/EthoML/VAME/pull/3
    * [x] Improve docstrings https://github.com/EthoML/VAME/pull/3
    * [x] Create API reference [@docs](/EthoML/VAME/tree/docs)
    * [x] FAQ section  [@docs](/EthoML/VAME/tree/docs)
    * [ ] Docs page about larger pipeline
    * [x] Add typehints https://github.com/EthoML/VAME/pull/3
* [x] Usability
    * [x] Improve progress information during training #3 
* [ ] Deprecate old repo

App:
* [ ] Continuous deployment for app creation
* [ ] Docs page for easy download
* [ ] Video walkthrough of demo

**Comment by luiztauffer** [comment]
closing as this is outdated now
